troubleshooting.md

```diff
@@ -47,4 +47,35 @@ communicate with a registry and not use tls verification.
 
   * Turn off tls verification by passing false to the tls-verification option.
   * I.e. `buildah push --tls-verify=false alpine docker://localhost:5000/myalpine:latest`
+
+---
+### 3) `buildah run` command fails with pipe or output redirection
+
+When doing a `buildah run` command while using a pipe ('|') or output redirection ('>>'),
+the command will fail, often times with a `command not found` type of error.
+
+#### Symptom
+When executing a `buildah run` command with a pipe or output misdirection such as the
```

Comment:
redirection not misdirection?

Reply:
Yes redirection is a better word.

Reply:
Oh shoot yes!  ty!

Reply:
Especially since the title has redirection, fixing.